packages/experiments/src/components/Pagination/Pagination.types.ts

```diff
@@ -0,0 +1,117 @@
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+
+export interface IPagination {}
+
+export interface IPaginationProps {
```

Comment:
Nit: keep the props alphabetized at least or have the related ones next to each other. Plus a little confused with so many `aria-label` props. We need to make sure they have good documentation and the naming is not confusing as to which element it applies.

Reply:
I tried to put the relevant ones together. Please see if its making more since now. In terms of the number of aria-label props, there are different buttons and we do need various props for the aria labels.

---
In reply to: [221850842](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r221850842) [](ancestors = 221850842)